theme/islandora-dublin-core-description.tpl.php

```diff
@@ -15,6 +15,6 @@
 <div class="islandora-metadata-sidebar">
   <?php if (!empty($dc_array['dc:description']['value'])): ?>
     <h2><?php print $dc_array['dc:description']['label']; ?></h2>
-    <p property="description"><?php print $dc_array['dc:description']['value']; ?></p>
+    <p property="description"><?php print nl2br(filter_xss($dc_array['dc:description']['value'])); ?></p>
```

Comment:
I think [what's been here, now](https://github.com/Islandora/islandora/pull/657/commits/2e24d92b5b7d451909da1d748c1ae958c1983607) is best... Seems kind of outside of the scope of this, but possibly throw in a ticket to drop the `filter_xss()` call from the DC class, since "working with [DC]" (the comment on the class) != "displaying in HTML"; however, would want to make sure that values are not being passed through raw anywhere...